redhat-upgrade-tool.spec

```diff
@@ -107,7 +107,7 @@ fi
 # boom doc files
 %license %{boom_dir}/COPYING
 %doc %{boom_dir}/README.md
-%doc %{_mandir}/man*/boom.*
+# %%doc %%{_mandir}/man*/boom.*
```

Comment:
Same as above:  Please keep the code clean.  There's no point in keeping comments that only clutter the file and add no value whatsoever.

Reply:
Agree. I am going to clean it.